--- src/Makefile
+++ src/Makefile
@@ -58,7 +58,7 @@ endif
 CFLAGS = -g -DXP_UNIX -Wall -DVERSION=$(VERSION)
 
 ifndef PYTHON
-  PYTHON = python
+  PYTHON = python2
 endif
 
 # Spidermonkey library.
@@ -135,5 +135,5 @@ install-pymod: pymod
 
 clean:
 	rm -f $(LIBRARY_LINK) $(LIBRARY) libjs.a pacparser.o pactester pymod/pacparser_o_buildstamp jsapi_buildstamp
-	cd pymod && python setup.py clean --all
+	cd pymod && $(PYTHON) setup.py clean --all
 	cd spidermonkey && $(MAKE) clean
